src/features/base-layout/components/footer/copyright.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @stylistic/jsx-one-expression-per-line */
```

Comment:
Do we need this?

Reply:
This is a temporary measure until we decide what we want to do with structures like this.

![image](https://github.com/rolling-scopes/site/assets/56927722/6a5d7aac-04e8-40de-9773-f479c9d28ffb)